src/Compiler/CompilationResult.php

```diff
@@ -0,0 +1,305 @@
+<?php
+
+/**
+ * SCSSPHP
+ *
+ * @copyright 2012-2020 Leaf Corcoran
+ *
+ * @license http://opensource.org/licenses/MIT MIT
+ *
+ * @link http://scssphp.github.io/scssphp
+ */
+
+namespace ScssPhp\ScssPhp\Compiler;
+
+use ScssPhp\ScssPhp\Util;
+use ScssPhp\ScssPhp\Exception\CompilerException;
+
+/**
+ * Compiler environment
+ *
+ */
+class CompilationResult
+{
+    protected static $resolvedImport = [];
+
+    protected $isCached = false;
+
+    /**
+     * @var string
+     */
+    protected $css = '';
```

Comment:
I suggest using `private` visibility in the new class, to avoid any BC needs on the properties.

Reply:
private on all the properties, fixed